Code Quality/Tools.md

```diff
@@ -6,51 +6,66 @@ If you have already read this section of the Handbook and are here just for the
 
 Git hooks allow us to run scripts during various `git` commands. This verification step can be used to run various tools which ensure that the code satisfies some conditions before it is added to the repository. If the verification fails, the git command will abort.
 
-No matter which code quality tools we use, git hooks are a great way to run those tools. There are multiple ways to add git hooks. For JavaScript projects, we recommend using [husky](https://github.com/typicode/husky). Husky scripts are configured in `package.json`. Here is a basic example which runs `test` before any code is pushed to the repository:
+No matter which code quality tools we use, git hooks are a great way to run those tools. There are multiple ways to add git hooks. For JavaScript projects, we recommend using [husky](https://github.com/typicode/husky).
 
-```js
-// package.json
-{
-  "husky": {
-    "hooks": {
-      "pre-push": "npm test"
-    }
-  }
-}
+Follow [usage](npm set-script prepare "husky install") guidelines for installing Husky and add hooks.
```

Comment:
I think you wanted to paste some URL to husky docs here, not some random command :D